src/components/CollectiveCard.js

```diff
@@ -317,9 +317,13 @@ class CollectiveCard extends React.Component {
                 <div className="role">{tierName}</div>
                 {membership.createdAt && (
                   <div className="since">
-                    <FormattedMessage id="membership.since" defaultMessage={'since'} />
-                    &nbsp;
-                    <FormattedDate value={membership.createdAt} month="long" year="numeric" />
+                    <FormattedMessage
+                      id="membership.since"
+                      defaultMessage={'since {date}'}
+                      values={{
+                        date: formatDate(membership.createdAt),
```

Comment:
Why not use FormattedDate?

Reply:
Unfortunately, `FormattedDate` can only be in English.

Reply:
That mat be a misconfiguration from us but it should normally localize dates. Do you have an example where it fails or a link to an issue?

Reply:
It has been reverted because it has nothing to do with this pull request modification.